src/pages/EnablePayments/AdditionalDetailsStep.js

```diff
@@ -385,8 +363,8 @@ export default compose(
             key: ONYXKEYS.WALLET_ADDITIONAL_DETAILS,
             initWithStoredValues: false,
         },
-        walletAdditionalDetailsDraft: {
-            key: ONYXKEYS.WALLET_ADDITIONAL_DETAILS_DRAFT,
```

Comment:
Looks like we are removing this subscription, but still referencing it in the code.

Reply:
Oh nevermind I see you are moving it to props now. Both work fine :D